src/Features/JsonPatch/src/Internal/ParsedPath.cs

```diff
@@ -14,8 +14,6 @@ namespace Microsoft.AspNetCore.JsonPatch.Internal
     /// </summary>
     public readonly struct ParsedPath
     {
-        private static readonly string[] Empty = null;
```

Comment:
This smells like a bug, are we sure this wasn't supposed to return `Array.Empty<byte>()`?

Reply:
Probably, but I don't really feel like digging up JsonPatch code 😄